model/src/main/kotlin/AnalyzerResult.kt

```diff
@@ -61,7 +61,7 @@ data class AnalyzerResult(
 
     fun collectErrors(): Map<String, List<String>> {
         val collectedErrors = mutableMapOf<String, MutableList<String>>()
-        collectedErrors["Analyzer"] = errors.toMutableList()
+        collectedErrors["${project.id.namespace}:${project.id.name}:${project.id.version}"] = errors.toMutableList()
```

Comment:
Why not `project.id.toString()`, which also includes the provider?

Reply:
Because the provider was added below, but I have refactored the whole method now.